content/docs/concepts/architecture/index.md

```diff
@@ -78,4 +78,4 @@ without having to authenticate. In this case, the distributor sidecars directly
 - When an execution plane is operated outside of the Cluster, it has the possibility to communicate with the HTTP API exposed by the `api-gateway-nginx`. In this case, every request to the API has to be authenticated using the `keptn-api` token. 
 Also, the distributor sidecars do not have the possibility to directly connect to the NATS cluster, but they can be configured to fetch open `.triggered` events from the HTTP API.
 
-To read more about developing execution plane services, please refer to the following section in the docs: [Write a Keptn-service](../../0.8.x/integrations/custom_integration/), which helps you to implement a custom service for Keptn. 
+To read more about developing execution plane services, please refer to the following section in the docs: [Write a Keptn-service](../../0.13.x/integrations/custom_integration/), which helps you to implement a custom service for Keptn. 
```

Comment:
```suggestion
To read more about developing execution plane services, please refer to the following section in the docs: [Write a Keptn-service](../../0.134x/integrations/custom_integration/), which helps you to implement a custom service for Keptn. 
```

Reply:
@StackScribe: can you please fix the ref by changing it to: `../../0.14.x`